### ISSUE TITLE
Accessibility/Identity Providers: User unable to select options from dropdown and options not read using tabs/arrows

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak-ui#3222]
### Describe the bug

The user is not able to select options from the dropdown and options are not read when using tabs/arrows.

This is not an issue for `Add  New Provider` in the User federation.

Tested on MacBook, Chrome browser using Chromevox.

### Version

19.0.0

### Expected behavior

All dropdown options should be accessible and read so that the user knows what they are before selecting.

### How to Reproduce?

- Click on Identity Providers from the left-side menu
-  Add a new identity provider
- Navigate to the `Add provider` dropdown and use Chromvox to listen to the options as you use tab/arrows to navigate between them
- Observe that tab/arrow navigation is not working and the options are not read
